### ISSUE TITLE
Selected labels are unselected when panning image using middle button

### ISSUE THREAD

**Issue by MichalLebeda** [heartexlabs/label-studio#3839]
**Describe the bug**
Selected labels are deselected when panning the image using mouse middle button

**To Reproduce**
1. select labels
2. drag by using middle mouse button
3. labels get unselected

**Expected behavior**
Keeping labels selected

**Environment (please complete the following information):**
 - OS: Win10, Edge
 - Label Studio Version: 1.7.1

**Comment by makseq** [comment]
Thank you for the report. [Ticket LSDV-4854](https://labelstudio.aha.io/develop/features/LSDV-4854) is created.
By the way, you can use hand tool (pressing `H`). Seems it doesn't unselect a label.

**Comment by MichalLebeda** [comment]
@makseq Yes, you are right. Its just hard to consciously press the H key when I am not used to it. The problem would probably be that middle click is registered as a regular click also
